templates/datetimepicker.mustache

```diff
@@ -0,0 +1,199 @@
+{{!
+@template block_evasys_sync/datetimepicker
+Example context (json):
+{
+}
+}}
+
+<select name="day_{{$id}}id{{/id}}" {{$disabled}}{{/disabled}}>
+        <option {{$day_1}}{{/day_1}}>1</option>
```

Comment:
What are the `{{$day_x}}{{/day_x}}`, `{{$month_y}}{{/month_y}}`, etc. for?

Reply:
You can set a standard value by setting this variable to selected in the including template